islandora.api.php

```diff
@@ -945,3 +945,24 @@ function callback_islandora_breadcrumbs_backends(AbstractObject $object) {
   // Do something to get an array of breadcrumb links for $object, root first.
   return array($root_link, $collection_link, $object_link);
 }
+
+/**
+ * Permit modules to alter the filename of a downloaded datastream.
+ *
+ * @param string $filename
+ *   The filename being created.
+ *
+ * @param AbstractDatastream $datastream
+ *   The datastream object being downloaded.
+ */
+function hook_datastream_filename_alter(&$filename, AbstractDatastream $datastream) {
```

Comment:
nice 👍

Reply:
@patdunlavey @d-r-p @rosiel just wondering now if the hook name should contain "islandora" in the name to make sure people know its something we provide and not a global Drupal thing? it is really just a convention thing, does not affect functionality but most of our hooks follow that naming

Reply:
You're right. I made the change. Thanks!